server/rucio.conf.j2

```diff
@@ -52,6 +52,9 @@ CacheRoot /tmp
 {% else %}
  SSLCACertificateFile /etc/grid-security/ca.pem
 {% endif %}
+ SSLCACertificatePath /etc/grid-security/certificates
```

Comment:
Please remove this and the following line. Both directives are already available and can be configured using the `RUCIO_CA_PATH` environment variable (c.f. lines 49-52). The certificates can already be mounted as you described and then `RUCIO_CA_PATH` can be used to point Apache to the mounted directory.

Reply:
Thomas, I applied the change.